app/assets/javascripts/application.js

```diff
@@ -1,2 +1,4 @@
 //= require turbolinks
-//= require app/utils
+
+import 'app/utils';
+import 'app/pw-toggle';
```

Comment:
if we didn't need a transpire step, I'd agree. To me, this is just as clear, and seems to make a bit more sense syntax-wise